### DISCUSSION TITLE
Encoded payloads: Is it possible with Avro Serializer to Save encoded messages in Kafka when sending it using AvroSerialize and doing the opposite when reading with Deserializer

### DISCUSSION THREAD

**Discussion by 1sumitkumar** [mateusjunges/laravel-kafka#288] [Q&A]
Currently I am using Avro Serializer and Deserializer and I am sending the Message object which is showing in the kafka cloud  like below but I want the message/object shown here to be encoded . Example something like "0000000001084a6f686e06446f6554"
. Can someone please help?

![image](https://github.com/mateusjunges/laravel-kafka/assets/163025449/d6280a67-38d5-4624-9e56-1fee65c8b3c3)

**Comment by mateusjunges** [comment]
I think the confluent cloud shows the message unserialized by default. Not sure if you can change that.

**Reply by 1sumitkumar** [reply]
@mateusjunges the message above is in binary bytes?
